client/src/routes/__layout.svelte

```diff
@@ -2,10 +2,31 @@
   import Breadcrumbs from "$lib/components/Breadcrumbs.svelte";
   import Header from "$lib/header/Header.svelte";
   import Footer from "$lib/footer/Footer.svelte";
+  import { onMount } from "svelte";
   import { page } from "$app/stores";
   import "../app.css";
+
+  // DSFR Assets
+  import appleTouchFavicon from "@gouvfr/dsfr/dist/favicon/apple-touch-icon.png";
+  import svgFavicon from "@gouvfr/dsfr/dist/favicon/favicon.svg";
+  import icoFavicon from "@gouvfr/dsfr/dist/favicon/favicon.ico";
+  import manifest from "@gouvfr/dsfr/dist/favicon/manifest.webmanifest";
+
+  onMount(async () => {
+    // Load the DSFR asynchronously, and only on the browser (not in SSR).
+    await import("@gouvfr/dsfr/dist/dsfr/dsfr.module.min.js");
```

Comment:
OK, ce script est bien intégré par Vite dans le build

```
[client] > svelte-kit dev
[client] 
[server] INFO:     Will watch for changes in these directories: ['/home/florimond/dev/prj-catalogue/catalogage-donnees/server']
[server] INFO:     Uvicorn running on http://127.0.0.1:3579 (Press CTRL+C to quit)
[server] INFO:     Started reloader process [189724] using watchgod
[client] Pre-bundling dependencies:
[client]   svelte
[client]   svelte/store
[client]   yup
[client]   @gouvfr/dsfr/dist/dsfr/dsfr.module.min.js
[client]   svelte/animate
[client]   (...and 4 more)
[client] (this will be run only when your dependencies or config have changed)
[client] 
[client]   SvelteKit v1.0.0-next.259
[client] 
...
```

Et en lançant `serve-dist` après le build on a des requêtes de ce type

```
GET /_app/chunks/dsfr.module.min-16d7900b.js
GET /_app/assets/favicon-43b3c676.svg
```

Donc tout est bien intégré. :+1:

Reply:
Et si je recharge la page (sans nuke le cache), les assets sont bien chargés avec une réponse `HTTP 304` (cache navigateur).

Par contre il y a toujours le "blink" / "flashing lights effects" (mon thème système est noir) avec rechargement complet de la page quand je clique par ex sur "Rechercher" puis sur le lien jumbotron. Faudra voir plus tard d'où ça vient. J'ai noté ça dans la fin de la description de #64.